### ISSUE TITLE
Is there a way to load a CSV file as a facts database (when using Tau Prolog under node)?

### ISSUE THREAD

**Issue by halloleo** [tau-prolog/tau-prolog#264]
In SWI-Prolog I have:

     :- use_module(library(csv)).

Is there anyting similar I can do in Tau Prolog (under node)?

Thanks for any pointers?

**Comment by pmoura** [comment]
Logtalk supports Tau Prolog and provides a `csv` library. But the this [bug](https://github.com/tau-prolog/tau-prolog/issues/265) is in the way of using it as it causes a compilation error.

**Comment by halloleo** [comment]
Thanks for this info. 

Do I understand it correctly that this is a general file-related bug  in Tau Prolog and not in LogTalk?

**Comment by pmoura** [comment]
Correct.

P.S. It's spelled "Logtalk", not "LogTalk".

**Comment by jariazavalverde** [comment]
No, there is no CSV library available in Tau Prolog. However, you can use the `js` package to call JavaScript functions from Prolog, so you can use a node.js package:

http://tau-prolog.org/documentation#js
